### DISCUSSION TITLE
Do paid Shodan keys provide any additional benefits over the free Shodan API key when using Prowler?

### DISCUSSION THREAD

**Discussion by j0eblow** [prowler-cloud/prowler#3800] [Q&A] [answered]
title is my question, thanks.

**Comment by jfagoagas** [comment, marked as answer]
Hi @j0eblow there is no difference beyond the API limits, we only use the Shodan service to check if an IP address configured in a Cloud resource is present in the Shodan database.

Thanks for using Prowler 🚀
